Dockerfile

```diff
@@ -21,6 +21,7 @@ COPY requirements-dev.txt .
 
 RUN pip install -r requirements.txt --force-reinstall sqlalchemy-filters
 RUN pip install -r requirements-dev.txt
+RUN pip install -r requirements-demo.txt
```

Comment:
I'm having demo requirements always installed (like dev requirements). Wanted to only have this installed when deploying the demo, but I think we're not setup for being so selective.